uprotocol/core/usubscription/v3/usubscription.proto

```diff
@@ -43,6 +43,14 @@ service uSubscription {
   option (version_minor) = 0;
   option (id) = 0;
 
+  // uSubscription change Notification topic that sends the Update message
+  option (notification_topic) = {
+    id: 0x8000,
+    name: "SubscriptionChange",
```

Comment:
the long URI will be core.udiscovery/1/SubscriptionChange#Update
Do we want to establish rules or convention for capitalization of the name ?

Reply:
Yes but not sure where we would document this, perhaps in the spec somewhere.